src/backend/data/models/user.js

```diff
@@ -37,6 +37,12 @@ export default function(sequelize, DataTypes) {
       type: DataTypes.BOOLEAN,
       defaultValue: false,
       field: 'is_admin'
+    },
```

Comment:
We will need to add this column to the model in production before deploying this.  Adding this as a reminder.